Backyarder/backyarder-ui
# Feature/cell actions

## What I did:
Cell action modal!!

###Testing notes:
- clicking on a cell that has a plant inside, you'll now see a modal just above with pic, name, and 3 buttons
- clicking the 'Plant!' button closes the modal and updates the `isPlanted` state to true and provides a green background around the cell indicating as such. 
- clicking the 'Remove' button removes the plant from the cell and closes the modal
  - ensure this works on a planted plant and an unplanted plant
- clicking the 'X' icon closes the modal with no other effect
- ensure disable functionality still works as intended

## Did this break anything?

## Type of change


## Checklist:

## What's next?
